src/effects/VST3/VST3OptionsDialog.h

```diff
@@ -0,0 +1,35 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file VST3OptionsDialog.h
+
+  @brief Part of Audacity VST3 module, duplicate from VSTEffect.cpp 
+
+**********************************************************************/
+
+#pragma once
+
+#include "widgets/wxPanelWrapper.h"
+#include "EffectHostInterface.h"
```

Comment:
The commit "Generator effect duration control added" doesn't compile.  The commit "User Presets" adds this include directive, which fixes it.  It would be nice to do an interactive rebase and make all the commits compile.